.github/workflows/sql-cli-test-and-build-workflow.yml

```diff
@@ -28,26 +34,16 @@ jobs:
           pip install -r requirements-dev.txt
           pip install setuptools wheel
       
-      #TODO: will setup CI for IT once we have OpenSearch and sql plugin release. Not it only runs UT
-      # It can also be refactored by launching OpenSearch instance with plugin installed from gradle.
-      
-      # - name: Set up ES and install SQL plugin
-      #   run: |
-      #     sudo add-apt-repository ppa:openjdk-r/ppa
-      #     sudo apt update
-      #     sudo apt install openjdk-14-jdk
-      #     sudo apt install unzip
-      #     wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.10.0-amd64.deb
-      #     sudo dpkg -i elasticsearch-oss-7.10.0-amd64.deb
-      #     sudo /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch https://d3g5vo6xdbdb9a.cloudfront.net/downloads/elasticsearch-plugins/opensearch-sql/opensearch_sql-1.12.0.0.zip
-      #     sudo systemctl start elasticsearch.service
+      - name: Download and run OpenSearch
+        run: |
+          wget -q https://artifacts.opensearch.org/releases/bundle/opensearch/2.2.1/opensearch-2.2.1-linux-x64.tar.gz
+          tar xf opensearch-2.2.1-linux-x64.tar.gz
+          opensearch-2.2.1/bin/opensearch-plugin remove opensearch-security
```

Comment:
Maybe a comment why we need to remove this plugin?

Reply:
Thank you, fixed in 1d935e6c.